deps/build.jl

```diff
@@ -73,30 +74,44 @@ mkpath(lib_dir)
 mkpath(bin_dir)
 
 
-function download_and_unpack(url)
-    tensorflow_zip_path = joinpath(base, "downloads/tensorflow.tar.gz")
-    download(url, tensorflow_zip_path)
+function download_and_unpack(url, checksum)
+    tensorflow_zip_path = joinpath(download_dir, "tensorflow.tar.gz")
+
+    use_cache = false
+    isfile(tensorflow_zip_path) && open(tensorflow_zip_path) do f
+        use_cache = bytes2hex(sha256(f)) == checksum
+    end
+        
+    if use_cache
+        @info "using cached $tensorflow_zip_path"
```

Comment:
unless the building fails, `@info` does not printing anything when `pkg> build TensorFlow`. Not sure if this is what we expect.